src/view/actions/sendEvent.jsx

```diff
@@ -92,44 +117,37 @@ const SendEvent = () => {
                 />
               </div>
             </div>
-            <div className="u-gapTop">
-              <InfoTipLayout tip="Influences whether the SDK should retrieve and render personalization content, among other things.">
-                <WrappedField
-                  name="viewStart"
-                  component={Checkbox}
-                  label="Occurs at the start of a view"
-                />
-              </InfoTipLayout>
-            </div>
             <div className="u-gapTop">
               <InfoTipLayout
-                tip="Please specify a data element that will return a JavaScript
-                  object in XDM format. This object will be sent to the Adobe
-                  Experience Platform."
+                tip="The type of the experience event.  Choose a predefined type or create
+                  your own. This will be added to the XDM object as the field `eventType`."
```

Comment:
I think users are probably not familiar with backticks. Maybe just `eventType` or `"eventType"`? I'm going to ask on Slack what others think, because this has come up before.

Reply:
An update: I asked in Slack and it was pretty inconclusive. It looks like we're using backticks in a few other places and were put there by more than one other developer, so I think we should go ahead with it unless you feel otherwise.